### PR TITLE
Add security.md file

### PR DESCRIPTION
**Pull request by riverlijunjie**
Details:
   - Add security.md file to this repo
  
Ticket:
   - 111140

### CODE REVIEW

**Review by nkogteva — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Security Policy
```
Comment (ilyachur) [inline SECURITY.md:1]: Do we need to add in README.md a link to this file?

Comment (riverlijunjie) [reply SECURITY.md:1]: It seems other repos don't add such link to this security file, such as openvino, opencv.
Do we need do it?

BTW, why it reports "Token Merging" fail, I don't have right to rebuild it, could you help re-build it? @ilyachur

Comment (ilyachur) [reply SECURITY.md:1]: I merged master branch to re-run all jobs

**Comment by riverlijunjie** [comment]
Could we merge this PR? @nkogteva @ilyachur

**Review by ilyachur — APPROVED** [review]

**Pull request merged**